source/includes/connection-snippets/x509-snippets/c-connection.c

```diff
@@ -0,0 +1,32 @@
+// begin x509 connection
+#include <mongoc/mongoc.h>
```

Comment:
@kevinAlbs Would it be prudent to also add `#include <stdio.h>` here as well? I think several of the libmongoc examples that print also include that.

Reply:
Good point, I think so. It's probably best not to rely on stdio.h being included through the include chain mongoc.h -> bson.h -> bson-compat.h.

Reply:
Fixed